spec/acceptance/postfix_exporter_spec.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 require 'spec_helper_acceptance'
+# puppet-postfix is not supported on Archlinux
+return if host_inventory['facter']['os']['name'] == 'Archlinux'
```

Comment:
I think the best practice is to use something like this:
```
describe 'prometheus postfix exporter', if: host_inventory['facter']['os']['name']!= 'Archlinux' do
```

Reply:
changed